tests/ui/pages/desktop/base.py

```diff
@@ -7,7 +7,7 @@ class Base(Page):
 
     _url = '{base_url}/{locale}'
 
-    def __init__(self, selenium, base_url, locale='en-US', **kwargs):
+    def __init__(self, selenium, base_url, timeout=60, locale='en-US', **kwargs):
```

Comment:
Since the guys don't want to switch to trusty&sudo I thought this would help with the slowness of the ci environment, but I don't think it actually does.